core/src/transport/choice.rs

```diff
@@ -51,13 +51,21 @@ where
         id: ListenerId,
         addr: Multiaddr,
     ) -> Result<(), TransportError<Self::Error>> {
+        println!("Trying address on {}...", std::any::type_name::<A>());
         let addr = match self.0.listen_on(id, addr) {
-            Err(TransportError::MultiaddrNotSupported(addr)) => addr,
+            Err(TransportError::MultiaddrNotSupported(addr)) => {
+                println!("Failed on {}", std::any::type_name::<A>());
```

Comment:
Here, I'd like debug level and also a message that includes address and transport.

Reply:
Done